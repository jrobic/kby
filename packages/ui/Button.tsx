import React from "react";

export const Button = () => {
  return <button type="button">Boop</button>;
};
