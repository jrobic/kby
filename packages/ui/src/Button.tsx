import { ReactElement } from "react";

export const Button = (): ReactElement => {
  return <button type="button">Boop</button>;
};
