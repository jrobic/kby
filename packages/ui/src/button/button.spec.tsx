import { render } from "@testing-library/react";
import { Button } from "./Button";

test("should render", () => {
  const { asFragment } = render(<Button label="Boop" />);

  expect(asFragment()).toMatchSnapshot();
});
