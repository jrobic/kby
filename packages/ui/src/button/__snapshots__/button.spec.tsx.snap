// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render 1`] = `
<DocumentFragment>
  <button
    class="storybook-button storybook-button--medium storybook-button--secondary"
    type="button"
  >
    Boop
  </button>
</DocumentFragment>
`;
